webapp/src/main/webapp/news-activity-composer-app/components/ExoNewsActivityComposer.vue

```diff
@@ -395,23 +395,29 @@ export default {
     editMode: function() {
       return this.activityId && this.activityId !== '';
     },
+    hasMediaContent() {
+      const parsedDocument = new DOMParser().parseFromString(this.news.body, 'text/html');
+      const mediaElements = parsedDocument.querySelectorAll('img, video, iframe');
```

Comment:
Unfortunately this is still insufficient as it is limited to those tags and does not cover all possible HTML tags that could be added (svg, canva, audio, etc ...) even if it not the case now, but everytime a new tag is allowed, we will have to update the fix.
I prefer doing comparison of raw HTML content